bindata/v4.1.0/aws-pod-identity-webhook/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-identity-webhook
+  namespace: openshift-cloud-credential-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pod-identity-webhook
+  template:
+    metadata:
+      labels:
+        app: pod-identity-webhook
+    spec:
+      serviceAccountName: pod-identity-webhook
+      containers:
+      - name: pod-identity-webhook
+        image: ${IMAGE}
+        imagePullPolicy: Always
+        command:
+        - /usr/bin/aws-pod-identity-webhook
+        - --in-cluster
+        - --namespace=openshift-cloud-credential-operator
+        - --port=6443
+        - --service-name=pod-identity-webhook
+        - --tls-secret=pod-identity-webhook
+        - --annotation-prefix=eks.amazonaws.com
```

Comment:
at some point before release, yes.  could we track this for a follow-on so that we aren't bogged down on naming; the hardest of issues in software development?

Reply:
agreed. maybe just stick a # TODO comment if possible.